poetry.lock

```diff
@@ -425,30 +425,43 @@ test-randomorder = ["pytest-randomly"]
 
 [[package]]
 name = "databricks-sql-connector"
-version = "3.7.1"
+version = "4.0.0b4"
```

Comment:
Do you know why this is still here?

Reply:
Yes, it's a transitive dependency https://github.com/databricks/databricks-sqlalchemy/blob/main/pyproject.toml#L13